### DISCUSSION TITLE
Creating custom page show error when go to role

### DISCUSSION THREAD

**Discussion by ayemin-tun** [filamentphp/filament#11877] [General]
Hello When I create custom page in filament and then I go to role detail it show that error why?

![image](https://github.com/filamentphp/filament/assets/76741291/bce46ba3-d745-4ec0-8cfd-4c7a53c0a07d)

**Comment by micraux** [comment]
Hello,

As far as I can see, the problem is with Filament Shield.

You may probably want to ask in [https://github.com/bezhanSalleh/filament-shield/issues](https://github.com/bezhanSalleh/filament-shield/issues)
